src/lib.rs

```diff
@@ -445,6 +452,33 @@ pub fn watchdog_enabled(unset_env: bool, usec: &mut u64) -> bool {
     }
 }
 
+fn monotonic_time_usecs() -> u128 {
+    use libc::{timespec, CLOCK_MONOTONIC, clock_gettime};
+
+    let mut monotonic_time = timespec {
```

Comment:
Let's use `MaybeUninit` like in https://docs.rs/nix/latest/src/nix/time.rs.html#191-198.

Reply:
Happy to do so, but seems more complicated when we can just init to 0. MaybeUninit also then comes with another unsafe dance after the call to gettime, when here we guarantee we are inited.